src/components/Toolbar/UpdateButton.js

```diff
@@ -1,7 +1,7 @@
-import { CircularLoader } from '@dhis2-ui/loader'
 import i18n from '@dhis2/d2-i18n'
 import { colors } from '@dhis2/ui-constants'
 import { IconSync16 } from '@dhis2/ui-icons'
+import { CircularLoader } from '@dhis2-ui/loader'
```

Comment:
Noticed that the import for CircularLoader is coming from a different dependency than lines 2 and 3: @dhis2-ui/loader vs @dhis2/ui-icons. Should they be the same?

Reply:
These should all be from the root package: `@dhis2/ui`